app/models/rake_ui/rake_task.rb

```diff
@@ -95,6 +95,7 @@ def call(args: nil, environment: nil, user_email: nil)
       )
 
       puts "[rake_ui] [rake_task] [forked] #{rake_task_log.rake_command_with_logging}"
+      upload_blank_log_file_on_s3(rake_task_log)
```

Comment:
Nit: Should we not simply pass `rake_task_log.log_file_name` instead of the object?